prep-html-syntax/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+  <title> Html Syntax exercise </title>
+<h1>
+<p style="color:red;">My favorite star wars character is Darth Vader! </p>
+  </h1>
```

Comment:
Hi Amir, the visible content such as `h1` or `p` should go within the `<body></body>` element of your document.